include/simdjson/generic/ondemand/json_iterator-inl.h

```diff
@@ -58,6 +58,30 @@ simdjson_warn_unused simdjson_really_inline error_code json_iterator::skip_child
       _depth--;
       if (depth() <= parent_depth) { return SUCCESS; }
       break;
+    case '"':
+      if(*peek() == ':') {
+        // we are at a key!!! This is
+        // only possible if someone searched
+        // for a key and the key was not found.
+        logger::log_value(*this, "key");
+        advance(); // eat up the ':'
+        break; // important!!!
+      }
+      // Because maniacs want to do crazy pedantic static analysis, we get the following
+      // error if we stop there...
+      // error: this statement may fall through [-Werror=implicit-fallthrough=]
```

Comment:
Another interesting way to deal with this is to use `continue` for the other cases.

Reply:
I'd love to know more about this continue trick, but I am going to resolve this issue with a new macro: simdjson_fallthrough.

Reply:
The continue trick would just be, use `continue` for everything except `"`, and then put the `default` case outside the switch. But I think finding ways to resolve it without this special case would be better anyway.